src/templates/articles/index.js

```diff
@@ -0,0 +1,175 @@
+    .map(a => a.article.data);
+  return (
+    <div className="container">
+      <div className="row article" role="main">
+        <Breadcrumbs links={[{ text: 'Статті', url: '/articles' }, { text: post.title }]} />
+        <article className="col-sm-12 col-lg-8" itemScope itemType="http://schema.org/Article">
+          <header>
+            <div className="article-category-container">
+              <Link to={`/categories/${post.category}`} className="square-link">
+                <span itemProp="articleSection">{post.category}</span>
+              </Link>
+            </div>
+            <h1 itemProp="headline">{post.title}</h1>
+            { post.subtitle && post.subtitle.length
+              ? <h2 className="subtitle" itemProp="description">{post.subtitle}</h2>
+              : null
+            }
+          </header>
+          {
+            post.image ?
+              <figure className="featured-image" itemProp="image" itemScope="" itemType="http://schema.org/ImageObject">
+                <img src={withPrefix(post.image)} alt={post.title} />
+                <meta itemProp="url" content={withPrefix(post.image)} />
+                <meta itemProp="width" content="1600" />
+                <meta itemProp="height" content="900" />
+              </figure> :
+              null
+          }
+          <div className="article-content" itemProp="articleBody" dangerouslySetInnerHTML={{ __html: post.html }} />
+          {
+            post.tags && post.tags.length ?
+              <div className="tags-container">Теги: {
+                (post.tags || []).map((tag, index) =>
+                  (<Link key={index} className="square-link" to={`/tags/${tag}`}>{tag}</Link>)) }
+              </div> :
+              null
+          }
+          <nav className="prev-next-article">
+            <div className="prev-next-inner">
+              <p className="prev-next-block-label">Читайте також</p>
+              <ul className="prev-next-links">
+                {
+                  post.previous ?
+                    <li className="prev-next-link-prev">
+                      <Link to={`/articles/${post.previous.path}`} rel="prev">
+                        <strong>Попередня стаття</strong>
+                        <span className="g1-gamma g1-gamma-1st">{post.previous.path}</span>
+                      </Link>
+                    </li> :
+                    null
+                }
+                {
+                  post.next ?
+                    <li className="prev-next-link-next">
+                      <Link to={`/articles/${post.next.path}`} rel="next">
+                        <strong>Наступна стаття</strong>
+                        <span className="g1-gamma g1-gamma-1st">{post.next.title}</span>
+                      </Link>
+                    </li> :
+                    null
+                }
+              </ul>
+            </div>
+          </nav>
+          { moreFromCategory.length ?
```

Comment:
length > 0
in React It will be true even if the array is empty

Reply:
No, it's ok. https://vpershe.netlify.com/articles/video-art/

Reply:
sorry, I haven't noticed you use the map... yeah, it's ok... you just don't display the items

Reply:
This check has nothing to do with map. In js and react [].length returns 0 and 0 is a falsy value.